src/Access/EnabledQuota.cpp

```diff
@@ -52,22 +48,24 @@ struct EnabledQuota::Impl
             return end;
         }
 
+        bool initial_end = !end_loaded.count(); /// whether the end of the interval wasn't calculated before
         const auto duration = interval.duration;
 
         do
         {
             end = end + (current_time - end + duration) / duration * duration;
```

Comment:
I don't understand what do you want to do in this line. `/` and `*` have the same priority.

Reply:
'/' and then '*' is a way to remove the remainder. I think I'll add a comment here.